network/calico.go

```diff
@@ -1,6 +1,54 @@
 package network
 
 func GetCalicoManifest(calicoConfig map[string]string) string {
+	awsIPPool := ""
+	if calicoConfig[CloudProvider] == AWSCloudProvider {
+		awsIPPool = `
+---
+kind: ConfigMap
```

Comment:
Is there way to not embed the yaml file contents in the calico.go? But rather store them in the separate yaml file, and just invoke it?